changelogs/DP-15605.md

```diff
@@ -0,0 +1,40 @@
+___EXAMPLE___
+Minor
+Added
+- (Patternlab) [Header] DP-4562: Set focus state for search on mobile menu in mobileNav module #473
+- (React) [HelpTip] DP-12875: Add `disabled` prop to disable HelpTip trigger text. #494
+
+___DESCRIPTION___
+Minor
+Changed
+- (Patternlab) [StickyToc] DP-15605: Add context info to sticky TOC buttons to reflect the state of the TOC display. Switch focus as the flyout toc shows/hides. #PR
```

Comment:
Can we add the PR # here?

```suggestion
- (Patternlab) [StickyToc] DP-15605: Add context info to sticky TOC buttons to reflect the state of the TOC display. Switch focus as the flyout toc shows/hides. #731 
```

Reply:
Thank you for pointing that out.  I forgot to do that.